aether-producer/conf/pip/primary-requirements.txt

```diff
@@ -17,6 +17,5 @@ flask
 gevent
 psycopg2-binary
 psycogreen
-requests
```

Comment:
If you change the primary requirements you should also change the frozen requirements.
`docker-compose -f docker-compose-connect.yml run --no-deps producer pip_freeze`

Reply:
I believe that `requests` is still required by `aether.client` so it's a dependency even though it's no longer a primary one.